components/ScreenOverlay.vue

```diff
@@ -48,7 +46,7 @@
     }
 
     .overlay__content {
-        width: 100%;
-        height: 100%;
+        height: calc(100dvh - 3.5rem);
```

Comment:
as 3.5 rem is defined as our Nav height, would it be better to have it in a SCSS variable and use it for calculation like this? 
Later on, if we want to change this we would have to manually change all calc

Reply:
Makes total sense